pghoard/common.py

```diff
@@ -142,26 +142,33 @@ def terminate_subprocess(proc, timeout=0.1, log=None):
     return proc.returncode
 
 
-def convert_pg_command_version_to_number(command_version_string):
+def extract_pg_command_version_string(command_version_string):
```

Comment:
Type hints would be nice here (and all following functions), especially the return type.